client/index.html

```diff
@@ -16,34 +16,35 @@
 </head>
 
 <body>
-  <article class="Article Article--intro">
-    <div class="Article--intro__header">
-      <h1 class="Article__h1">
-        <img src="./view/Article/browserlist_logo.svg" alt="">
-        Browserslist
-      </h1>
-      <a class="Badge" href="https://github.com/browserslist/browserslist">
-        <img src="./view/Badge/github_logo.svg" alt="Count of github stars" class="Badge__githubLogo"> 10.5k
-      </a>
-      <a class="Badge" href="https://twitter.com/Browserslist">
-        <img src="./view/Badge/twitter_logo.svg" alt="Count of twitter followers"> 4k
-      </a>
-    </div>
-    <p class="Article--intro__about">Shared browser and platform compatibility config for popular JavaScript tools,
-      including Autoprefixer, Babel, ESLint, PostCSS, and Webpack</p>
-    <p class="Article--intro__supported">
-      Supported by
-      <img src="./view/Article/evil_maritians_logo.svg" alt="">
-      <a href="https://evilmartians.com/" class="Link" target="_blank">Evil Martians</a> and
-      <img src="./view/Article/cube_logo.svg" alt="">
-      <a href="https://cube.dev/" class="Link" target="_blank">Cube</a>
-    </p>
-  </article>
-
-  <article class="Article Article--stats" data-id="query_container">
-    <h2>Display compatible browsers</h2>
-    <form data-id="query_form" class="Form">
-      <div class="Form__queryTextAreaContainer">
+  <main>
```

Comment:
Why we need to wrap all content to `main`? Should `main` be about the main content excluding footer and etc?

Reply:
What do you think if we will wrap form into `<main>`? It is the main content of the website.

`min` is used to fast travel, it will be nice to have fast-travel to form.

Reply:
> What do you think if we will wrap form into <main>? It is the main content of the website.
> min is used to fast travel, it will be nice to have fast-travel to form.

it seems good, but 2 articles will be without a parent, which is a bit strange

let me make a separate issue about this

> min is used to fast travel, it will be nice to have fast-travel to form.

I like the idea of `go to form`

Reply:
@ariarzer 

For now, remove `<main>`. Let's talk about it in a separate Issue :)
https://github.com/browserslist/browsersl.ist/issues/389


And in general, PR is difficult to read because of this.

Reply:
@ariarzer 

For now, remove `<main>`. Let's talk about it in a separate Issue :)
https://github.com/browserslist/browsersl.ist/issues/389


And in general, PR is difficult to read because of this.